oxidation/bindings/generator/generate.py

```diff
@@ -175,7 +179,11 @@ class RefTypeInUse(BaseTypeInUse):
 class StrBasedTypeInUse(BaseTypeInUse):
     kind = "str_based"
     name: str
+    type: str
+    namespace: str
 
+    # The error returned when trying to parse the value from a string.
+    from_str_error: str
```

Comment:
I don't see the need for this given we already specified in `custom_from/to_rs_string` the return type is AsRef<str>

Reply:
We could however add typing info in the `custom_from/to_rs_string`:

```rust
    let custom_from_rs_string: Fn(String) -> Result<{{ type.rs_type }}, AsRef<str>> = {{ type.custom_from_rs_string }};
    let value: Result<{{ type.rs_type }}, _> = custom_from_rs_string({{ js_val }}.value({{ mut_cx_ref }}));
```